src/components/ActivitiesList.js

```diff
@@ -13,6 +13,10 @@ const ActivitiesList = ({ activities }) => (
   </div>
 );
 
+ActivitiesList.defaultProps = {
+  activities: []
+};
+
 ActivitiesList.propTypes = {
   activities: PropTypes.array
```

Comment:
I would be more specific. Sorry, that I haven't given this hint before. 
Imo, we should tell more about array and. 
I think that you have already `activityPropTypeShape`. So... this is an `arrayOf(activityPropTypeShape)`